src/pevent_lib.cc

```diff
@@ -55,11 +58,34 @@ static void display_system_config(void) {
   }
 }
 
+int pevent_compute_min_mmap_order(int min_buffer_size_order,
```

Comment:
@nsavoire @sanchda would you agree with computing the size of the ring buffer based on the requested sample size ?

Reply:
ok for me